Services/sdLitica.Services/Management/Exceptions/EmailUsedException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace sdLitica.Services.Management.Exceptions
+{
+    public class EmailUsedException : BaseExceptionModel
```

Comment:
Not to have million exceptions, we can make this one more generaic, for example PropertyDuplicationException with two parameters 'propName' and 'propValue' and mesage like

Property '{propName}' value '{propValue}' already exists in the system

Or something like this

Reply:
Great!